data/logs_model/logs_producer/splunk_logs_producer.py

```diff
@@ -22,19 +23,45 @@ def __init__(
         url_scheme="https",
         verify_ssl=True,
         index_prefix=None,
+        ssl_ca_path=None,
```

Comment:
What's the path of this parameter, could you pls give an example?

Reply:
`ssl_ca_path` is not mandatory but is required along with `verify_ssl: True`. You would set it to the relative container path where your cert is stored. Ex: `conf/stack/<cert-name>`